src/main/steps/c100-rebuild/miam/miam-info/content.ts

```diff
@@ -0,0 +1,122 @@
+  legalAidLink:
+    '<a href="https://www.gov.uk/check-legal-aid" class="govuk-link" target="_blank" aria-label="Check if you’re eligible for legal aid during mediation">Check if you’re eligible for legal aid during mediation</a> - welsh',
+  miamMeeting:
+    'A <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> is a one-off meeting and not the same as mediation. - welsh',
+  miamHeading1: 'What happens at a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamHeading2:
+    'If you’ve already attended a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamHeading3:
+    'When you don’t have to attend a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamList1: 'how mediation works - welsh',
+  miamList2: 'the benefits of mediation - welsh',
+  miamList3: 'whether mediation is right for you - welsh',
+  miamList4: 'the likely costs - welsh',
+  miamList5: 'if you may qualify for help with the costs of mediation and legal advice - welsh',
+  miamList6: 'other options you could use to help you reach an agreement - welsh',
+  line1:
+    'At the <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> a mediator will explain: - welsh',
+  line2:
+    'Your mediator will also discuss the <a href="https://www.gov.uk/guidance/family-mediation-voucher-scheme" class="govuk-link" target="_blank" aria-label="Family Mediation Voucher Scheme">Family Mediation Voucher Scheme</a> with you if you are eligible. - welsh',
+  line3Link:
+    '<a href="https://www.familymediationcouncil.org.uk/find-local-mediator/" class="govuk-link" target="_blank" aria-label="Find a mediator to book a MIAM">Find a mediator to book a MIAM</a> - welsh',
+  line4:
+    'You should have a document signed by the mediator confirming this. You need to upload this document to your application. - welsh',
+  line5:
+    'You do not have to attend a MIAM if you have a valid reason. For example, you or the children are at risk of harm. - welsh',
+  line6Link:
+    '<a href="https://www.gov.uk/check-legal-aid" class="govuk-link" target="_blank" aria-label="List of valid reasons for not attending a MIAM">List of valid reasons for not attending a MIAM</a> - welsh',
+  miamLabel: 'Attending a MIAM - welsh',
+  miamConsentStatement:
+    'I understand that I have to attend a MIAM or provide a valid reason for not attending. - welsh',
+  errors: {
+    miamConsent: {
+      required:
+        'Confirm that you understand that you have to attend a MIAM or provide a valid reason for not attending. - welsh',
+    },
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    miamConsent: {
```

Comment:
you will have to rename the fields to miam_

Refer to this PR - https://github.com/hmcts/prl-citizen-frontend/pull/351/files